__tests__/suggestions.test.js

```diff
@@ -206,4 +206,19 @@ describe('Suggestions', function() {
 
     expect($el.componentDidUpdate.called).to.equal(true);
   });
+
+  test('should render custom suggestions', function() {
+    const $el = shallow(
+      mockItem({
+        renderSuggestion: ({ text }) => (
+          <div>
+            <i />
+            {text}
+          </div>
+        ),
+      })
+    );
+
+    expect($el.find('i').length).to.equal(4);
```

Comment:
instead checking `i` can you add a class to the div in line 214 and check then assert with classname here

Reply:
done